src/Sylius/Bundle/ApiBundle/Resources/config/api_platform/resources/AdminUser.xml

```diff
@@ -42,6 +42,26 @@
                     </values>
                 </normalizationContext>
             </operation>
+            <operation class="ApiPlatform\Metadata\Put" uriTemplate="admin/administrators/{id}.{_format}">
+                <denormalizationContext>
+                    <values>
+                        <value name="groups">
+                            <values>
+                                <value>sylius:admin:admin_user:update</value>
+                            </values>
+                        </value>
+                    </values>
+                </denormalizationContext>
+                <normalizationContext>
+                    <values>
+                        <value name="groups">
+                            <values>
+                                <value>sylius:admin:admin_user:show</value>
+                            </values>
+                        </value>
+                    </values>
+                </normalizationContext>
+            </operation>
```

Comment:
Why do you need a PUT for the admin user in this PR? 😅

Reply:
That was tricky 😄  Some `managing_product` scenarios require changing admin locales.

Reply:
These scenarios should probably wait for implementing admin user endpoints 😅, but as it was already done, it's ok 👌🏻